src/styl/_helpers/_colors.styl

```diff
@@ -14,6 +12,29 @@ $helpers-colors ?= merge($colors-themes, {
     highlight: $color-highlight,
 })
 
+.exampleBlock
```

Comment:
Ho no ! If you need some css for demo, prefer inline style in `colors.twig`.
We have to keep in mind that CSS will be used for some apps and we don't want to load useless css (and it is aggravated for AMP).